atlantis.yaml

```diff
@@ -0,0 +1,29 @@
+version: 2
+
+projects:
+
+- name: "tfstate-backend"
+  workflow: "make"
+  dir: "conf/tfstate-backend"
+  workspace: "default"
+  terraform_version: "v0.11.10"
+  autoplan:
+    when_modified:
+      - "Makefile"
+      - "*.tf"
+      - "*.tfvars"
+      - ".envrc"
+    enabled: true
+  apply_requirements:
+    - "approved"
+
+workflows:
+  make:
+    plan:
+      steps:
+      - run: "export | grep TF_"
+      #- run: "direnv exec . init-terraform"
+      #- run: "direnv exec . make plan"
```

Comment:
Commented code should not be committed. I do not know the intent of this change. _why_ has this been commented out?

Reply:
agree - was just debugging/triaging. getting back to this now.